src/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ target_link_libraries(qmedia
     PUBLIC
         ${LIBRARIES}
         OpenSSL::Crypto
-        openh264
+        ${openh264}
```

Comment:
@paulej  does this variable gets defined for non windows platforms.

Reply:
It should.  This is coming via vcpkg.